orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -133,3 +133,7 @@ def buffers(self, recurse: bool = True) -> Iterator[torch.Tensor]:
     def named_buffers(self, prefix: str = '', recurse: bool = True) -> Iterator[Tuple[str, torch.Tensor]]:
         """Override original method to delegate execution to the base module"""
         yield from self._flattened_module._base_module.named_buffers(prefix=prefix, recurse=recurse)
+
+    def _replicate_for_data_parallel(self):
```

Comment:
Add docstring which explains why it is incompatible and list the workaround. It will be part of public documentation

Reply:
That can be present in the PR description too